docs/PSConsoleHostReadline.md

```diff
@@ -0,0 +1,50 @@
+---
+ms.date: 12/07/2018
+schema: 2.0.0
+locale: en-us
+keywords: powershell,cmdlet
+external help file: PSReadLine-help.xml
+title:  PSConsoleHostReadLine
+---
+
+# PSConsoleHostReadLine
+
+## SYNOPSIS
+This function is the main entry point for PSReadLine.
+
+## SYNTAX
+
+```
+PSConsoleHostReadLine
+```
+
+## DESCRIPTION
+
+`PSConsoleHostReadLine` is the main entry point for the PSReadLine module. The PowerShell console
+host automatically loads the PSReadLine modules and calls this function. Under normal operating
+conditions, this function is not intended to be used from the command line.
+
+If you wanted to create your own custom ReadLine function, you could replace this function with
```

Comment:
I wouldn't really say here "if you wanted to create your own ..." but you could refine the description so that is inferred.

This module doesn't enable creating your own custom ReadLine function, it is a custom ReadLine function.

The extension point `PSConsoleHostReadLine` is special to the console host - it will call any alias/function/script with this name. PSReadLine defines this function so that it is called from the console host.

Reply:
I will rework this description and submit an update.